tardis/utilities/asyncbulkcall.py

```diff
@@ -125,7 +125,7 @@ async def _bulk_dispatch(self):
             # track tasks via strong references to avoid them being garbage collected.
             # see bpo#44665
             self._bulk_tasks.add(task)
-            task.add_done_callback(lambda _: self._bulk_tasks.discard(task))
+            task.add_done_callback(lambda _, task=task: self._bulk_tasks.discard(task))
```

Comment:
During development the newest flake version complaint about B023 in this line. So, as agreed on the this PR, I have added a fix.